packages/assets-controllers/src/index.ts

```diff
@@ -8,4 +8,4 @@ export * from './TokenDetectionController';
 export * from './TokenListController';
 export * from './TokenRatesController';
 export * from './TokensController';
-export { formatIconUrlWithProxy } from './assetsUtil';
+export { formatIconUrlWithProxy, getFormattedIpfsUrl } from './assetsUtil';
```

Comment:
This has been extracted to https://github.com/MetaMask/controllers/pull/973 and will remain here until that's merged.